content/how-to/cloud-backup/rackspace-cloud-backup-backup-actions/index.md

```diff
@@ -24,17 +24,17 @@ You can access the **Backup Actions** menu in the following ways:
     on the gear icon next to the backup name on the **Configured Backups**
     page.
 
-    {{<image src="ScreenShot2015-10-15at4.20.11PM.png" alt="" title="">}}
+    {{<image src="ScreenShot2022-08-09at4.20.11PM.png" alt="" title="">}}
```

Comment:
rename image as well. I also see an IP Address here. So remove or replace if it is a security risk.

Reply:
resolved